packages/client-search/src/methods/index/saveSynonyms.ts

```diff
@@ -10,14 +10,15 @@ export const saveSynonyms = (base: SearchIndex) => {
     synonyms: readonly Synonym[],
     requestOptions?: SaveSynonymsOptions & RequestOptions
   ): Readonly<WaitablePromise<SaveSynonymsResponse>> => {
-    const { forwardToReplicas, replaceExistingSynonyms, ...options } = requestOptions || {};
+    const { forwardToReplicas, clearExistingSynonyms, replaceExistingSynonyms, ...options } =
+      requestOptions || {};
     const mappedRequestOptions = createMappedRequestOptions(options);
 
     if (forwardToReplicas) {
       mappedRequestOptions.queryParameters.forwardToReplicas = 1; // eslint-disable-line functional/immutable-data
     }
 
-    if (replaceExistingSynonyms) {
+    if (replaceExistingSynonyms || clearExistingSynonyms) {
       mappedRequestOptions.queryParameters.replaceExistingSynonyms = 1; // eslint-disable-line functional/immutable-data
```

Comment:
Is it normal that you kept `replaceExistingSynonyms` in `mappedRequestOptions.queryParameters` ?

Reply:
the actual query parameter name is still replaceExistingSynonyms @yannickcr